indexing-service/src/test/java/org/apache/druid/indexing/common/task/IndexTaskTest.java

```diff
@@ -2604,6 +2613,81 @@ private static IndexIngestionSpec createIngestionSpec(
     }
   }
 
+  @Test
+  public void testAuthorizeRequest() throws Exception
+  {
+    // Need to run this only once
+    if (lockGranularity == LockGranularity.SEGMENT) {
```

Comment:
This is a parameterized unit test. So every test method is run once for SEGMENT granularity and once for TIME_CHUNK. Since this test does not depend on granularity, figured we should run it only once.

We could explain this part better in the comment or just remove the check.
Please let me know what you think would be better.